app/boards/shields/architeuthis_dux/architeuthis_dux_left.conf

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2021 The ZMK Contributors
```

Comment:
We usually add one single .conf file, not left and right, since that's the file our setup scripts look for.

Reply:
Got it, thanks! The left and right .conf files were from previously distributing the shield in a `zmk-config`-style repo. I've now removed them from this PR.